### PR TITLE
Updated Dashboard UI

### PR DESCRIPTION
**Pull request by popQA17**
**Please describe the changes this PR makes and why it should be merged:**

This PR finally updated the Homepage landing, by including sections such as:
- Home
- Features
- Call-To-Action

and updated UI for the dashboard Sidebar.

**Status and versioning classification:**

- This PR changes the library's interface (methods or parameters added)
- This PR includes breaking changes (methods removed or renamed, parameters moved or removed)

### CODE REVIEW

File: dashboard/pages/index.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const Home = (_props: any) => {
               marginTop: '1rem'
           }}>Login
             </Button>
-            <Button color="primary" flat onClick={()=> router.push('/login')} style={{
+            <Button color="primary" flat onClick={()=> window.open('https://github.com/SudhanPlayz/Discord-MusicBot')} style={{
```
Comment (DarrenOfficial) [inline dashboard/pages/index.tsx:42]: shouldn't this be pointed to /login?

Comment (SudhanPlayz) [reply dashboard/pages/index.tsx:42]: Yea it should be pointed to `/login`

Comment (DarrenOfficial) [reply dashboard/pages/index.tsx:42]: oh

**Comment by SudhanPlayz** [comment]
image
Thanks for the contribution!

**Pull request merged**